### ISSUE TITLE
unable to adjust initial learning rate in v1

### ISSUE THREAD

**Issue by thegooby** [MIC-DKFZ/nnUNet#1463]
Hello, I've not been able to adjust the number of epochs or initial learning rate in the original nnunet. I've looked through the files that the trainerv2 pulls from but only the trainer itself seems to define the initial learning rate. I checked the suggested location to find learning rate variant files but what I'm looking for isn't there. As an example, I can put the initial learning rate to 0.0001 and nothing changes. I can even comment out the self.initial_lr line or set it to 0 and the console will still show the initial learning rate to be 0.01.  Relevant code snippet below:

   ```
def __init__(self, plans_file, fold, output_folder=None, dataset_directory=None, batch_dice=True, stage=None,
                 unpack_data=True, deterministic=True, fp16=False):
        super().__init__(plans_file, fold, output_folder, dataset_directory, batch_dice, stage, unpack_data,
                         deterministic, fp16)
        self.max_num_epochs = 1000
        self.initial_lr = 0.0001
        self.deep_supervision_scales = None
        self.ds_loss_weights = None

        self.pin_memory = True
```
For some additional information, the package I am trying to use is this one https://github.com/JunMa11/SegLoss specifically the focal loss. I understand that I should be able to make a custom trainer class to get pass unexpected behavior, but even with the focal loss trainer here and adding a line to define the learning rate just like above, the initial learning rate stays at 0.01.
